db/seeds/seed.js

```diff
@@ -116,4 +116,4 @@ const seed = ({ topicData, userData, articleData, commentData }) => {
     });
 };
 
-module.exports = seed;
+module.exports = { seed };
```

Comment:
be careful adding this here, it might effect other files that use it (in thinking the run-seed file specifically :) )

Reply:
Hi Jim, my tests fail when I change this back?